attestation-agent/attestation-agent/src/eventlog.rs

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) 2024 Alibaba Cloud
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+use std::{fs::File, io::Write};
+
+use anyhow::{Context, Result};
+
+/// AA's eventlog will be put into this path
+pub const EVENTLOG_PATH: &str = "/run/attestation-agent/eventlog";
+
+pub const EVENTLOG_PARENT_DIR_PATH: &str = "/run/attestation-agent";
```

Comment:
this can also be determined runtime using `.parent()` from `Path` with `EVENTLOG_PATH`.

Reply:
Right. As they two are hardcoded a const would be better

Reply:
sounds good. Let me add a fix